web/src/components/storage/AutoSizeText.tsx

```diff
@@ -175,7 +175,7 @@ function AutoSizeTextDynamic({
         // TRANSLATORS: %1$s is a mount point and %2$s is a list of other paths
         return sprintf(
           _(
-            "The size range for %1$s will be dynamically adjusted based on the amount of RAM in the system, the usage of Btrfs snapshots and the presence of separate file systems for %2$s.",
+            "The size for %1$s will be dynamically adjusted based on the amount of RAM in the system, the usage of Btrfs snapshots and the presence of separate file systems for %2$s.",
```

Comment:
NP: Not sure if we should take the opportunity for clarifying translator the subtle difference between this translation and the one in singular form.